nucleus/common/internal-api/src/main/java/org/glassfish/internal/deployment/DeploymentTracing.java

```diff
@@ -83,7 +88,19 @@
 
     }
 
-    public static enum ModuleMark {
+    public enum AppStage {
```

Comment:
This would need bit of cleanup, some of the "stages" can be merged and component used to distinguish them, but we need to collect more diverse traces yet to decide on that.